bs-web-app/src/components/NavBar.tsx

```diff
@@ -12,13 +38,30 @@ export default function NavBar(): JSX.Element {
           <Typography variant="h2" sx={{ flexGrow: 1 }}>
             🏝 Brokensea
           </Typography>
-          <Button variant="outlined" onClick={() => setIsOpen(true)}>
-            Connect Wallet
-          </Button>
-          <ConnectWalletModal
-            isOpen={isOpen}
-            handleClose={() => setIsOpen(false)}
-          />
+          {isLoggedIn ? (
+            <Chip
+              avatar={
+                <Avatar
+                  style={{
+                    color: "green",
+                  }}
+                ></Avatar>
+              }
+              label={signerAddress}
```

Comment:
We don't have to do it on this PR, but most dapps will have a shortened version of a user's address. A good future feature. Some examples:

app.sushi.com
<img width="400" alt="Screen Shot 2021-10-09 at 6 30 39 PM" src="https://user-images.githubusercontent.com/8534926/136675784-41a36a86-7d3c-423c-8140-b19c873ecd29.png">
etherscan.io
<img width="251" alt="Screen Shot 2021-10-09 at 6 30 08 PM" src="https://user-images.githubusercontent.com/8534926/136675785-56f68e2d-b00c-4137-b6bd-853d802fde0c.png">
matcha.xyz
<img width="252" alt="Screen Shot 2021-10-09 at 6 29 57 PM" src="https://user-images.githubusercontent.com/8534926/136675786-7dbea0b6-9d76-422c-b744-9a417073d44a.png">


Another future feature: display ENS name here if a user has it

Reply:
Yep that's the plan.